packages/cli/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## `7.10.4`
```

Comment:
Please change the header to `## Recent Changes`. Our deployment automation will handle updating the version number when a new version is published.